dev/scripts/components/CardGridPage.js

```diff
@@ -127,8 +127,9 @@ class CardGridPage extends React.Component {
       : cardSet = this.state.allCardsInSet;
     }
 
-    return (
-      <React.Fragment>
+    let emblemType = ["colorless","lightning", "fighting", "grass", "fire", "psychic", "fairy", "metal", "water"]
```

Comment:
- no global variables plz. What is the best place for this data to live, keeping in mind that we want to be able to use CardGridPage for ALL franchises and the emblemTypes associated with each franchise?
- We don't ever reassign this variable, so no need to define using `let`